packages/modules/providers/file-s3/src/services/s3-file.ts

```diff
@@ -99,6 +99,7 @@ export class S3FileService extends AbstractFileProviderService {
       // protected private_secret_access_key_: string
 
       // ACL: options.acl ?? (options.isProtected ? "private" : "public-read"),
+      ACL: "public-read",
```

Comment:
should this be `options.acl ?? (options.isProtected ? "private" : "public-read")` ?

Reply:
@carlos-r-l-rodrigues we haven't added support for private files yet, so I'm leaving it hard-coded until we do.